src/components/Human.vue

```diff
@@ -44,6 +44,13 @@ export default {
     position: relative;
     margin: 0 40px;
 
+    &:hover {
+      .human__left-arm {
+        transform-origin: 20% 40%;
```

Comment:
Does this need a comment for hard-coded values?

Reply:
I'm going to let it slide because currently the project is only used by me (and it's your first ever PR) 😅